addons/comments-linebreaks/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Linebreaks in comments",
-  "description": "Shows the linebreaks (pressing enter) in comments.",
+  "description": "Shows the linebreaks (pressing enter) in comments on the website.",
```

Comment:
Should it be `Enter`?

Reply:
And I'm pretty sure it should be `line breaks`, 2 words.

Reply:
`Enter` and `linebreaks`

Reply:
Google returns 478,000 returns for `linebreak` and 3,990,000 results for `line break`

Reply:
Something like *Makes line breaks (pressing Enter) in comments on the website visible instead of converting them to spaces* might be better than the current description.